napari_btrack/_tests/test_dock_widget.py

```diff
@@ -32,79 +33,92 @@ def test_add_widget(make_napari_viewer):
         widget_name="Track",
     )
 
-    assert len(list(viewer.window._dock_widgets)) == num_dw + 1  # noqa: S101
+    assert len(list(viewer.window._dock_widgets)) == num_dw + 1
 
 
 @pytest.fixture
 def track_widget(make_napari_viewer) -> Container:
```

Comment:
Should we move the `pytest.fixture`s to a `conftest.py` in case there are more tests in future?

Reply:
yeah probably. There are some other changes that could be made to the tests, like using the builtin `tmp_path` fixture. Maybe we should use a separate pr for these things?

Reply:
Have raised an issue #65